<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gpAnalizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Application</string>
    </property>
    <addaction name="newApp"/>
    <addaction name="deleteApp"/>
    <addaction name="copyApp"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New </string>
   </property>
  </action>
  <action name="newApp">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="deleteApp">
   <property name="text">
    <string>Delete current</string>
   </property>
  </action>
  <action name="copyApp">
   <property name="text">
    <string>Copy current</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
